### DISCUSSION TITLE
CORS policy issue during web3auth modal initialization

### DISCUSSION THREAD

**Discussion by Myrindd** [Web3Auth/Web3Auth#1119] [Help]
Hi everyone
I am currently trying to implement the modal version of web3auth on a cross-origin-isolated website. During the initialization of the interface, the component called "sdk-modal" fails to download due to a lack of cross-origin-resource-policy in the header. Is there any way around that issue?

**Comment by shahbaz17** [comment]
Related to https://github.com/orgs/Web3Auth/discussions/1127

**Comment by Myrindd** [comment]
![image](https://user-images.githubusercontent.com/67431109/209697824-23ba0fb8-e728-4903-b137-be36d0135726.png)

**Reply by Myrindd** [reply]
Potential fix: set the cross-origin-resource-policy in the response header to:
Cross-Origin-Resource-Policy: cross-origin
